app/controllers/hyrax/generic_works_controller.rb

```diff
@@ -204,22 +204,23 @@ def download
   def globus 
     require 'tempfile'
 
-    # hard coding the nectar umrdr-data directory to
-    # use unless ENV['GLOBUSDIR'] is set
-    globus_dir = ENV['GLOBUSDIR'] || "/hydra-dev/umrdr-data/globus"
+    globus_dir = Umrdr::Application.config.globus_dir
     # globus_dir = "."
     folder = globus_dir + "/DeepBlueData_" + curation_concern.id
+    Rails.logger.debug "Globus begin copy to folder #{folder}"
     #use dot folders to flag download status
-    complete_folder = folder + "/.complete"
+    complete_folder = folder + "/#{Umrdr::Application.config.globus_complete}"
```

Comment:
These path-building expressions should be done with `Pathname` (or `File` / `Dir`). Anywhere we run the app will generally tolerate `/` as directory separator, but that is not really the point here. The point is that having a proper object affords convenience without resorting to string smashing. For example:

```ruby
globus_dir = Pathname.new(Umrdr::Application.config.globus_dir)
folder = globus_dir + "DeepBlueData_#{curation_concern.id}"
complete_folder = folder + Umrdr::Application.config.globus_complete
```

I'm unaware of anything in the standard library that won't happily accept a `Pathname` anywhere it would accept a `String`. If some third party method requires a string, `.to_s` and `.to_path` are equivalent (and used by library methods like `File.open` when passed a Pathname).

See also: https://ruby-doc.org/stdlib-2.4.0/libdoc/pathname/rdoc/Pathname.html